batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentController.java

```diff
@@ -94,17 +99,19 @@ private void createProcess(HttpServletRequest request, HttpServletResponse respo
     String name = request.getParameter("name");
     String address = request.getParameter("address");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "address" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)